### PR TITLE
Booking Ajax

### PR DESCRIPTION
**Pull request by thanhsangcao**

### CODE REVIEW

**Comment by thanhsangcao** [comment]
ready

File: app/Http/Controllers/Site/SiteController.php [hunk 1]
```diff
@@ -12,89 +12,48 @@
...
-        $booking = $booking->create([
+        $booking = Booking::create([
             'salon_id' => $request['salon_id'],
             'status' => trans('booking.nobook'),
             'name' => $request['name'],
             'phone_number' => $request['phone_number'],
-            'time_booking' => $request['timesheetstylist'],
+            'time_booking' => $request['timesheet'],
             'stylist_id' => $request['stylist_id']
         ]);
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php:39]: đưa vào repository đi nhé

File: app/Http/Controllers/Site/SiteController.php [hunk 2]
```diff
@@ -12,89 +12,48 @@
...
-        $timesheetstylist = TimeSheetStylist::all();
-        $salon = Salon::where('id', '<>', 1)->get();
-        $user = User::where('permission', 2)->get();
-
-        return view('sites.booking2', compact('user', 'salon', 'timesheetstylist'));
+        $salons = Salon::where('id', '<>', 1)->get();
+        $selectSalon[null] = trans('main.select');
+        foreach($salons as $salon) {
+            $selectSalon[ $salon->id ] = $salon->name . ' - ' . $salon->address;
+        }
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php:24]: đưa vào repository đi nhé

File: app/Http/Controllers/Site/SiteController.php [hunk 3]
```diff
@@ -12,89 +12,48 @@
...
         ]);
-        $booking->save();
 
         return redirect('/')->with('status', trans(''));
     }
+    
+    public function getStylist(Request $request){
+        $salon_id = $request->salon_id;
+        $users = User::where('salon_id', $salon_id)->get();
+        $view = view("sites.chooseStylist",compact('users'))->render();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php]: convention
tên view
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4

File: app/Http/Controllers/Site/SiteController.php [hunk 4]
```diff
@@ -12,89 +12,48 @@
...
+
+        return response()->json(['html'=>$view]);
+    }
+
+    public function getTimesheet(Request $request){
+        $stylist_id = $request->stylist_id;
+        $timeSheet = TimeSheetStylist::where('stylist_id', $stylist_id)->first();
+        $view = view("sites.chooseTimeSheet",compact('timeSheet'))->render();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php]: convention
tên view
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4

File: public/js/pagination.js [hunk 5]
```diff
@@ -14,4 +14,58 @@ $(document).ready(function(e) {
...
+    });
+
+    $('#salon_id').on('change', function (e) {
+        $('.timesheet').empty();
+        var salon_id = this.value;
+        $.ajax({
+            type : "post",
+            url : "getStylist",
+            data : { salon_id : salon_id },
+            dataType : "json",
```
Comment (BunnyPi04) [inline public/js/pagination.js]: sử dụng nháy đơn
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12

File: resources/views/home.blade.php [hunk 6]
```diff
@@ -84,50 +83,34 @@
...
-                                        var page = $(this).attr('href').split('page=')[1];
-                                        
-                                        $.get('product?page=' + page, function(data) {
-                                            $('body').html(data);
-                                        }); 
+                                </tr>
+                            </thead>
+                            <tbody>
+                            @foreach ($users as $user)
+                                @if($user->permission =='2')
```
Comment (BunnyPi04) [inline resources/views/home.blade.php]: convention
hằng số đưa vào config

File: resources/views/sites/booking.blade.php [hunk 7]
```diff
@@ -1,36 +1,129 @@
...
-                    {{ Form::label('', null, ['class' => 'text-success form-name']) }}
-                        <h4>
-                            {{ trans('booking.input') }}
-                        </h4>
-                            <div class="form-group">
-                                {!! Form::label('', __('Name')) !!}
-                                {{ Form::text('name', '', ['class' => 'name-input form-control input-lg', 'placeholder' => trans('booking.name'), 'required' => 'required']) }}
+                        <div class="wizard-navigation">
+                            <ul>
+                                <li><a href="#info" data-toggle="tab">Step 1: Infomation</a></li>
```
Comment (BunnyPi04) [inline resources/views/sites/booking.blade.php]: i18n

File: resources/views/sites/booking.blade.php [hunk 8]
```diff
@@ -1,36 +1,129 @@
...
-                                {!! Form::label('', __('Phone')) !!}
-                                {{ Form::text('phone_number', '', ['class' => 'phone-input form-control input-lg', 'placeholder' => trans('booking.phone'), 'required' => 'required']) }}
+                        </div>
+                        <div class="wizard-footer">
+                            <div class="pull-right">
+                                <input type='button' class='btn btn-next btn-fill btn-info btn-wd' name='next' value='Next' />
+                                {!! Form::button('finish', ['type' => 'submit', 'class' => 'btn btn-finish btn-fill btn-danger btn-wd', 'name' => 'finish']) !!}
+                            </div>
+                            <div class="pull-left">
+                                <input type='button' class='btn btn-previous btn-fill btn-default btn-wd' name='previous' value='Previous' />
```
Comment (BunnyPi04) [inline resources/views/sites/booking.blade.php]: sử dụng laravel collective

**Comment by BunnyPi04** [comment]
commented

File: app/Http/Controllers/Site/SiteController.php [hunk 9]
```diff
@@ -12,89 +12,52 @@
...
+    public function getStylist(Request $request){
+        $salon_id = $request->salon_id;
+        $users = User::where('salon_id', $salon_id)->get();
+        $selectStylist[null] = trans('main.select');
+        foreach ($users as $user) {
+            $selectStylist[ $user->id ] = $user->name;
+        }
+        $view = view("sites.choose_stylist",compact('selectStylist','users'))->render();
+
+        return response()->json(['html'=>$view]);
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php]: convention

File: app/Http/Controllers/Site/SiteController.php [hunk 10]
```diff
@@ -12,89 +12,52 @@
...
     }
+    
+    public function getStylist(Request $request){
+        $salon_id = $request->salon_id;
+        $users = User::where('salon_id', $salon_id)->get();
+        $selectStylist[null] = trans('main.select');
+        foreach ($users as $user) {
+            $selectStylist[ $user->id ] = $user->name;
+        }
+        $view = view("sites.choose_stylist",compact('selectStylist','users'))->render();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php]: convention

File: app/Http/Controllers/Site/SiteController.php [hunk 11]
```diff
@@ -12,89 +12,52 @@
...
+
+        return response()->json(['html'=>$view]);
+    }
+
+    public function getTimesheet(Request $request){
+        $stylist_id = $request->stylist_id;
+        $timeSheet = TimeSheetStylist::where('stylist_id', $stylist_id)->first();
+        $view = view("sites.choose_timesheet",compact('timeSheet'))->render();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Site/SiteController.php]: convention

File: resources/views/master.blade.php [hunk 12]
```diff
@@ -4,17 +4,32 @@
...
     {{ Html::style('bower_components\bootstrap\dist\css\bootstrap.css') }}
     {{ Html::style('css\style.css') }}
+    Multi-Column-Select
+    {{ Html::style('bower_components\bower-hair\Multi-Column-Select\Multi-Column-Select.css') }}
+    <!--     Fonts and icons     -->
+	{{ Html::style('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons') }}
+	{{ Html::style('https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css') }}
 
+	<!-- CSS Files -->
+	{{ Html::style('bower_components/bower-hair/booking/css/bootstrap.min.css') }}
```
Comment (BunnyPi04) [inline resources/views/master.blade.php]: tab lại

File: resources/views/master.blade.php [hunk 13]
```diff
@@ -4,17 +4,32 @@
...
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name = "csrf-token" content="{{ csrf_token() }}"/>
     {{ Html::style('bower_components\bootstrap\dist\css\bootstrap.css') }}
     {{ Html::style('css\style.css') }}
+    Multi-Column-Select
+    {{ Html::style('bower_components\bower-hair\Multi-Column-Select\Multi-Column-Select.css') }}
+    <!--     Fonts and icons     -->
+	{{ Html::style('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons') }}
+	{{ Html::style('https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css') }}
```
Comment (BunnyPi04) [inline resources/views/master.blade.php]: ko sử dụng link online
tải package về

**Comment by BunnyPi04** [comment]
commented

**Comment by thanhsangcao** [comment]
ready

**Pull request merged**